frontend/src/components/create-user-page/CreateUser.tsx

```diff
@@ -1,3 +1,118 @@
-import React from "react";
+import React, { FormEvent, useContext, useState } from "react";
+import { LoginContext } from "../login/LoginManager";
 
-export const CreateUser: React.FunctionComponent = () => <></>;
+export const CreateUser: React.FunctionComponent = () => {
+  const loginContext = useContext(LoginContext);
+
+  const [formData, setFormData] = useState({
+    name: "",
+    username: "",
+    email: "",
+    password: "",
+  });
+  const validatePassword = (passwordToCheck: string) => {
+    const upper = /(.*[A-Z].*)/;
+    const lower = /(.*[a-z].*)/;
+    const number = /[0-9]/;
+    const special = /[-+_!@#$%^&*.,?]/;
+
+    if (passwordToCheck.length < 8) {
+      alert("Please make sure password is longer than 8 characters.");
+      return false;
+    }
+
+    if (!upper.test(passwordToCheck)) {
+      alert("Please make sure Password Includes an UpperCase character");
+      return false;
+    }
+
+    if (!lower.test(passwordToCheck)) {
+      alert("Please make sure Password Includes a LowerCase character");
+      return false;
+    }
+
+    if (!number.test(passwordToCheck)) {
+      alert("Please make sure Password Includes a Digit");
+      return false;
+    }
+
+    if (!special.test(passwordToCheck)) {
+      alert(
+        "Please make sure Password Includes a special character from -+_!@#$%^&*.,?"
+      );
+      return false;
+    }
+  };
+
+  const handleSubmit = (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    if (
+      formData.name === "" ||
+      formData.username === "" ||
+      formData.email === "" ||
+      formData.password === ""
+    ) {
+      alert("All fields required!");
```

Comment:
As above, avoid alerts - prefer to add text below the specific missing field that says "this field is required" (but, again, you can save that for the follow-up styling ticket)

Reply:
So, for this do we need to use useState and set some text for the required field and style it?

Reply:
We will resolve this when we do styling ticket 16.